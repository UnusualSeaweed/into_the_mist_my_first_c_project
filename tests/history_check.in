place A1
place B1
place A2
place B2
place A3
place B3
place A4
place B4
place C1
place D1
place C2
history
