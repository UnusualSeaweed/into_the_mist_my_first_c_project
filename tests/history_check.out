A1B1A2B2A3B3A4B4C1D1C2
